src/libsync/owncloudpropagator.cpp

```diff
@@ -599,6 +680,16 @@ bool PropagateDirectory::scheduleNextJob()
     if (_state == NotYetStarted) {
         _state = Running;
 
+        if(_bundleJob){
+            PropagateBundle* bundle = dynamic_cast<PropagateBundle*>(_bundleJob.take());
```

Comment:
For security agains memory leaks. I am anyways removing this scoped pointer later and changing the parent to _subJobs